Makefile

```diff
@@ -9,14 +9,10 @@ WATCH = "\
 			require('shelljs').exec(cmd) 										  \
 		})"
 
-
-jshint:
-	# check code quality
-	@ ./node_modules/jshint/bin/jshint lib test
-
-jscs:
+eslint:
 	# check code style
-	@ ./node_modules/jscs/bin/jscs lib test
+	# TODO: Get ./test up to standards and add back
```

Comment:
This TODO is here because some PRs slipped through without being checked. It should be a small fix to get it back in line but it is outside of the scope of this PR. If this gets merged I will follow up with this work.